### PR TITLE
CO-2417_CoGroup_Select_action_should_enforce_copersonid_named_parameter_presence

### PR DESCRIPTION
**Pull request by ioigoume**
Default to current user if copersonid named param is missing for CoGroups Select action

### CODE REVIEW

File: app/Controller/CoGroupsController.php [hunk 1]
```diff
@@ -753,7 +753,9 @@ public function paginationConditions() {
     // First get the CoPersonID to work against either the current user, or the user being acted on in select mode
     $coPersonId = $this->Session->read('Auth.User.co_person_id');
     if($this->action == 'select') {
```
Comment (spartanicus) [inline app/Controller/CoGroupsController.php:755]: For simplicity, might we just add the test for a non-empty coPersonId here?

Comment (ioigoume) [reply app/Controller/CoGroupsController.php]: Done
